src/module/item/condition/document.ts

```diff
@@ -132,13 +132,15 @@ class ConditionPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends
 
         if (this.system.persistent) {
             const { dc, damageType } = this.system.persistent;
+            const rollOptions = this.getRollOptions("recovery");
```

Comment:
Not sure if I should be using getRollOptions() here, or if I should just add the ones I need specifically.